blueflood-core/src/integration-test/java/com/rackspacecloud/blueflood/io/TestIO.java

```diff
@@ -0,0 +1,24 @@
+package com.rackspacecloud.blueflood.io;
+
+/**
+ * Utility class to provide some test methods existing tests use.
+ */
+public interface TestIO {
```

Comment:
It's a helper class for testing to be implemented by Astyanax & Datastax.  TestHelperIO?